src/make.js

```diff
@@ -0,0 +1,19 @@
+module.exports = function make(...numb) {
+    if (typeof numb[0] == "number") {
+        let arr = [...numb];
```

Comment:
You don't need to create a new array `arr`, you could use `numb`, it is already an array